tests/sentry/feedback/usecases/test_create_feedback.py

```diff
@@ -546,5 +546,91 @@ def dummy_response(*args, **kwargs):
                 .new_status
                 == GroupStatus.RESOLVED
             )
+
         if not (expected_result and feature_flag):
             assert mock_produce_occurrence_to_kafka.call_count == 1
+
+
+@django_db_all
+def test_create_feedback_spam_detection_option_false(
+    default_project,
+    mock_produce_occurrence_to_kafka,
+    monkeypatch,
+):
+    default_project.update_option("sentry:feedback_ai_spam_detection", False)
```

Comment:
nice, looks like this tests if the proj option is False/off then we won't be running the spam detection on it👍